Source/towners.cpp

```diff
@@ -57,13 +57,10 @@ void InitTownerInfo(int i, const TownerData &townerData)
 	townerData.init(towner, townerData);
 }
 
-void LoadTownerAnimations(Towner &towner, const char *path, int frames, Direction dir, int delay)
+void LoadTownerAnimations(Towner &towner, const char *path, int frames, int delay)
 {
 	towner.data = LoadFileInMem(path);
-	for (auto &animation : towner._tNAnim) {
-		animation = towner.data.get();
-	}
-	NewTownerAnim(towner, towner._tNAnim[dir], frames, delay);
+	NewTownerAnim(towner, towner.data.get(), frames, delay);
 }

```

Comment:
Towner::_TNAnim was only ever used as an array in InitCows. For every other towner every index has the same pointer to the sprite sheet and the only read was in this function. Just referencing the data pointer directly saves some dead code and removes the implication that direction actually does something.